.github/workflows/staging.yaml

```diff
@@ -32,6 +30,8 @@ jobs:
         uses: crazy-max/ghaction-docker-meta@v1
         with:
           images: ${{ env.DOCKER_IMAGE }}
+          tags: |
+            type=semver,pattern={{raw}}
```

Comment:
Can you add a comment either here, or in the readme, about how the tagging now works?

It seems there's still no per-commit tags (I now see `staging`, `features-tags`, `gh_tags` and `staging_tags`), but no commit ID, so it doesn't solve #14 yet.

Reply:
the image now looks like:
quay.io/wire/outlook-addin:staging@sha256:63a90c4a70a4bdbad6e9e4166db88c2b5a163005797cd76fc822e222a1b245b9

Reply:
This still means it's called `staging`, the part behind is just the sha256 content hash that this tag currently has.

Reply:
I dont know how to fix it. I am just a simple Java developer :'(

Reply:
I think we want to use `type=sha,format=long` (https://github.com/docker/metadata-action#typesha) generally, and in addition `type=ref` (https://github.com/docker/metadata-action#typeref), at least for builds on master and staging branches (maybe PRs too).